<launch>

    <arg name="grasp_rob1_ip"/>
    <arg name="grasp_rob1_state_port"/>
    <arg name="grasp_rob1_motion_port"/>
	<arg name="grasp_rob1_motion_syncer_port"/>
	<arg name="toggle_grippers_during_regrasp"/>

    <!-- <node name="Grasping_Robot_1_State" pkg="robots_server" type="grasping_robot1_state_node" 
    	args="$(arg grasp_rob1_ip)
			$(arg grasp_rob1_state_port)"/> -->
    
	<node name="Grasping_Robot_1_Motion" pkg="robots_server" type="grasping_robot1_motion_node" output="screen"
		args="$(arg grasp_rob1_ip)
			$(arg grasp_rob1_motion_port)
			$(arg toggle_grippers_during_regrasp)"/>
   
   <node name="Grasping_Robot_1_Motion_Syncer" pkg="robots_server" type="grasping_robot1_motion_syncer_node" output="screen"
		args="$(arg grasp_rob1_ip)
			$(arg grasp_rob1_motion_syncer_port)"/>
   
</launch>